src/common/CMakeLists.txt

```diff
@@ -7,6 +7,8 @@
 
 set(TARGET aoscommoncpp)
 
+option(USE_INTERNAL_MBEDTLS "Use internal mbedtls library" OFF)
```

Comment:
`aoscommoncpp` should be built without link with mbedtls. It is up to client (zephyr app) to provide mbedtls. So, it should be moved to tests cmake I guess.